.github/workflows/build-one.yml

```diff
@@ -84,10 +86,17 @@ jobs:
         run: |
```

Comment:
Is a clippy run supposed to be exclusive to 'ubuntu-latest' (not also 'oxide-colo-builder')?

Reply:
we could add it but it's not strictly necessary. I forget why we restricted it to linux only.